### PR TITLE
Cleaner

### PR DESCRIPTION
**Pull request by NacymeG**

### CODE REVIEW

**Review by John-athan14 — DISMISSED** [review]

File: src/Components/MagicCard/MagicCard.jsx [hunk 1]
```diff
@@ -3,16 +3,14 @@ import PropTypes from 'prop-types';
...
-      <img src={imageUrl} alt={name} />
-      <button className="myButton" type="button">
-        Description
-      </button>
-      <button className="myButton" type="button">
-        Add to Deck
-      </button>
-    </figure>
+    <>
+      {imageUrl ? (
```
Comment (Hacene010) [inline src/Components/MagicCard/MagicCard.jsx]: did this code (imageUrl &&...) not work ?

File: src/Components/SectionCaroussel/SectionCaroussel.jsx [hunk 2]
```diff
@@ -1,20 +1,38 @@
...
+    axios
+      .get(
+        `https://api.magicthegathering.io/v1/cards?random=true&set=MID&page=1&pageSize=100`
+      )
+      .then(({ data }) => {
+        setCardLists(data.cards);
+      });
+  }, []);
+
+  // const result = cardLists.filter((card) => card.imageUrl);
```
Comment (Hacene010) [inline src/Components/SectionCaroussel/SectionCaroussel.jsx]: dead code here. 
Remove it plz

**Review by Hacene010 — CHANGES_REQUESTED** [review]
some changes to make

**Review by Hacene010 — APPROVED** [review]
seems ok. 
Didn't see that your SM didn't review it before

**Review by John-athan14 — APPROVED** [review]

**Pull request merged**